spec/tanker/identity_spec.rb

```diff
@@ -9,11 +9,13 @@
     @app = test_app
     @user_id = 'b_eich'
     @user_email = 'brendan.eich@tanker.io'
+    @hashed_email = Base64.strict_encode64(Tanker::Crypto.generichash(@user_email, Tanker::Identity::BLOCK_HASH_SIZE))
```

Comment:
Nitpick: don't you want a test vector here too, instead of recomputing the value?

Reply:
I'm not sure actually, but there you go!